app/services/person_constructor.rb

```diff
@@ -58,24 +63,21 @@ def flatten_remote_fields
   end
 
   def strip_whitespace_from_values
-    params.merge!(params){ |k, v1| v1.try(:strip) || v1 }
+    params.merge!(params){ |k, v| v.try(:strip) || v }
```

Comment:
I think changing to use `v` instead of `v1` is fine here, but just FYI: the reason it was `v1` before was to make it clear that we're keeping the value from the first hash and to imply that there's a v2 that we're ignoring (calling `merge` with a block gives you three arguments to work with: `key`, `value1` (from first hash), and `value2` (from second hash).